rllib/evaluation/worker_set.py

```diff
@@ -788,147 +790,17 @@ def session_creator():
             logger.debug("Creating TF session {}".format(config["tf_session_args"]))
             return tf1.Session(config=tf1.ConfigProto(**config["tf_session_args"]))
 
-        def valid_module(class_path):
```

Comment:
All moved into `RolloutWorker` (only used there anyways!). Arguably could also be moved into AlgorithmConfig itself (`AlgorithmConfig.make_input|output_creator()`)?